### PR TITLE
Correct some typos and make small improvements on GettingStarted.mo

### PR DESCRIPTION
**Pull request by max-privato**
Fix some typos and make other improvements on gettingStarted.mo

### CODE REVIEW

File: OpenIPSL/UsersGuide/GettingStarted.mo [hunk 1]
```diff
@@ -4,44 +4,44 @@ model GettingStarted "Getting Started"
 
   annotation(DocumentationClass=true, Documentation(info="<html>
 <p>
-The <strong><code>OpenIPSL</code></strong> was developed to be a familiar alternative to traditional
+The <strong><code>OpenIPSL</code></strong> was developed to be a familiar alternative to traditional
```
Comment (dietmarw) [inline OpenIPSL/UsersGuide/GettingStarted.mo:7]: Unnecessary trailing white space introduced.

File: OpenIPSL/UsersGuide/GettingStarted.mo [hunk 2]
```diff
@@ -4,44 +4,44 @@ model GettingStarted "Getting Started"
...
+The <strong><code>OpenIPSL</code></strong> was developed to be a familiar alternative to traditional 
 power system analysis tools.
 There are, however, some particularities to the nature of the library, presented in this Section.
 </p>
 <h4>Power flow and Initialization</h4>
 <p>
-All models in a Modelica library require initial guess values that should come from a solution of
-the steady state of the overall model. This initial guess is called power flow.
-However, there is no power flow solver associated to the library as of the present time.
+All models in a Modelica library require initial guess values that should come from a solution of
```
Comment (dietmarw) [inline OpenIPSL/UsersGuide/GettingStarted.mo:13]: whitespace

File: OpenIPSL/UsersGuide/GettingStarted.mo [hunk 3]
```diff
@@ -4,44 +4,44 @@ model GettingStarted "Getting Started"
...
 There are, however, some particularities to the nature of the library, presented in this Section.
 </p>
 <h4>Power flow and Initialization</h4>
 <p>
-All models in a Modelica library require initial guess values that should come from a solution of
-the steady state of the overall model. This initial guess is called power flow.
-However, there is no power flow solver associated to the library as of the present time.
+All models in a Modelica library require initial guess values that should come from a solution of 
+the steady state of the overall model. This initial guess in case of power system studies is commonly the result of the so-called power flow analysis.
+However, there is no power flow solver associated to the library as of the present time.
```
Comment (dietmarw) [inline OpenIPSL/UsersGuide/GettingStarted.mo:15]: whitespace

File: OpenIPSL/UsersGuide/GettingStarted.mo [hunk 4]
```diff
@@ -4,44 +4,44 @@ model GettingStarted "Getting Started"
...
 such as <a href=\"https://github.com/SanPen/GridCal\">GridCal</a> or <a href=\"https://github.com/FRESNA/PyPSA\">PyPSA</a>.
 </p>
 <p>
 From these values, a Modelica tool solves the initialization problem for all algebraic and differential - state variables.
-All models in <strong><code>OpenIPSL</code></strong> are programmed in such way that by introducing a <a href=\"https://github.com/SmarTS-Lab/Raw2Record\">power flow solution</a> (from another tool), the <strong>initial guesses</strong> are computed as parameters within each model and are provided into the initial equations that are used to solve the overall initialization problem.
-See <a href=\"http://www.ep.liu.se/ecp/article.asp?issue=119&amp;article=010\">this paper</a> for a more detailed explanation.
+All models in <strong><code>OpenIPSL</code></strong> are programmed in such way that by introducing a <a href=\"https://github.com/SmarTS-Lab/Raw2Record\">power flow solution</a> (from another tool), the <b>initial guesses</b> are computed as parameters within each model and are provided into the initial equations that are used to solve the overall initialization problem.
```
Comment (dietmarw) [inline OpenIPSL/UsersGuide/GettingStarted.mo:31]: Replacing `&amp;` will cause issues.

File: OpenIPSL/UsersGuide/GettingStarted.mo [hunk 5]
```diff
@@ -4,44 +4,44 @@ model GettingStarted "Getting Started"
...
-In a effort to harmonized the presentation of the power flow parameters, a model should extend <strong>pfComponent</strong> that will provide all the necessary parameters for data coming from power flow solutions to be used in the initialization of the model.
-The <strong>pfComponent</strong> also provide the support for the common <strong>SystemBase</strong> component that provides a single instance of the system's common parameters, i.e., frequency and base power.
+As we have just mentioned, the full initialization of the components&apos; internal variables and states is achieved by a set of internal <b>initial equations</b> that are to be derived by the developer of the model.
+In a effort to harmonize the presentation of the power flow parameters, a model should extend <b>pfComponent</b> that will provide all the necessary parameters for data coming from power flow solutions to be used in the initialization of the model.
+The <b>pfComponent</b> also provides support for the common <b>SystemBase</b> component that in turn provides a single instance of the system's common parameters, i.e., frequency and base power.
```
Comment (dietmarw) [inline OpenIPSL/UsersGuide/GettingStarted.mo:37]: don't use `b`, `strong` is the correct one.

**Review by dietmarw — CHANGES_REQUESTED** [review]

**Comment by max-privato** [comment]
I'm rather new to pull requests discussions.
I agree to correct spaces and substitute /b with /strong.
I don't see where &amp was used, but I understand that this is a technical change, and therefore I agree with your suggestion regarding it.

Unfortunately, I don't know if I have to do anything to accept these changes, and how to do this...
I would be glad if you do these small changes yourself.

**Comment by dietmarw** [comment]
Since the fixes are so minor and isolating the typo fixes from the wrong syntax changes takes far more time to solve I'm going to close this PR unmerged. The getting started guide will most likely get updated again anyway. I hope you understand. Feel free to attempt another PR with clean changes that don't contain unwanted changes that would also break CI.

**Comment by max-privato** [comment]
> Since the fixes are so minor and isolating the typo fixes from the wrong syntax changes takes far more time to solve I'm going to close this PR unmerged. The getting started guide will most likely get updated again anyway. I hope you understand. Feel free to attempt another PR with clean changes that don't contain unwanted changes that would also break CI.

Sure I understand.
Frankly I don't understand what caused the wrong syntax, so that to propose clean changes.
I don't remember exactly  but I used dymola or openmodeloca to implement the proposed changes.

**Pull request closed without merging**